pom.xml

```diff
@@ -115,11 +115,11 @@
       -->
 
       <activemq-surefire-argline>-Djava.util.logging.manager=org.jboss.logmanager.LogManager
-         -Dlogging.configuration=file:${activemq.basedir}/tests/config/logging.properties
-         -Djava.library.path=${activemq.basedir}/artemis-native/bin/ -Djgroups.bind_addr=localhost -Dorg.apache.activemq.artemis.api.core.UDPBroadcastEndpointFactory.localBindAddress=localhost
+         -Dlogging.configuration='file:${activemq.basedir}/tests/config/logging.properties'
+         -Djava.library.path='${activemq.basedir}/artemis-native/bin/' -Djgroups.bind_addr=localhost -Dorg.apache.activemq.artemis.api.core.UDPBroadcastEndpointFactory.localBindAddress=localhost
          -Djava.net.preferIPv4Stack=true
       </activemq-surefire-argline>
-      <activemq.basedir>${project.basedir}</activemq.basedir>
+      <activemq.basedir>${basedir}</activemq.basedir>
```

Comment:
Agreed.  However, when I didn't use the consolidated variable, and instead let it recompute each time, I had to quote on each declaration.  It wasn't pretty.  I can switch to that if 3.0 compatibility is a must.